server/db/models/Orders.js

```diff
@@ -0,0 +1,19 @@
+const Sequelize = require("sequelize");
+const db = require("../db");
+
+const Order = db.define("order", {
+  date: {
+    type: Sequelize.DATE,
+    allowNull: false,
+  },
+  shippingInfo: {
```

Comment:
instead of shipping/billing, maybe add name/last/name/email/address/creditcard information